src/olympia/addons/tests/test_views.py

```diff
@@ -5598,7 +5598,9 @@ def test_prevent_too_complex_to_determinize_exception(self):
         for i in range(0, 10):
             addon_factory()
         self.refresh()
-        query = '남포역립카페추천 ˇjjtat닷컴ˇ ≡제이제이♠♣ 남포역스파 남포역op남포역유흥≡남포역안마남포역오피 ♠♣'
+        query = (
+            '남포역립카페추천 ˇjjtat닷컴ˇ ≡제이제이♠♣ 남포역스파 남포역op남포역유흥≡남포역안마남포역오피 ♠♣'  # noqa: E501
+        )
```

Comment:
This (and the other 2 changes) are due to https://github.com/charliermarsh/ruff/issues/3825 where ruff's line length is counted by unicode width rather than string length.  However, when I added the `(` and/or split the string over multiple lines Black would auto-format them back again. I will follow-up in that issue - as alignment with Black was one of the justifications - but in the meantime the line exclusion works.

Reply:
fwiw, tl;dr from https://github.com/charliermarsh/ruff/issues/3825 is that Black only supports calculating by unicode width in preview mode - and it's currently buggy - so we'll likely need the line exclusion until at least the start of 2024 when the feature moves out of preview in Black.